src/common/utils/match-route-order-independent/index.tsx

```diff
@@ -0,0 +1,326 @@
+import { pathToRegexp } from 'path-to-regexp';
```

Comment:
Should we use `.ts` file extension for files in this directory?

Reply:
ah, good point. Let me change it to `.ts`.